docs/user-guide/node-api.md

```diff
@@ -35,7 +35,7 @@ This can be useful, for example, when making a text editor plugin that passes in
 
 A [stylelint configuration object](configuration.md).
 
-If no `config` or `configFile` is passed, stylelint will look for a `.stylelintrc` configuration file.
+If no `config` or `configFile` is passed, stylelint will use a [config lookup algorithm](https://stylelint.io/user-guide/configuration/#loading-the-configuration-object) to find the correct config.
```

Comment:
Please use a relative path here, so that the link works within the docs of the repo i.e. `./configuration.md#loading-the-configuration-object`. This change means the link will also be linted by `remark-lint`, ensuring it is always valid.